client/me/help/help-contact/index.jsx

```diff
@@ -211,16 +210,14 @@ module.exports = React.createClass( {
 	},
 
 	onOperatorsAway: function() {
-		const { isOlarkReady, isUserEligible, details } = this.state.olark;
-		const showChatVariation = isUserEligible && details.isOperatorAvailable;
-		const showKayakoVariation = ! showChatVariation && ( details.isConversing || isUserEligible );
-		const showForumsVariation = ! ( showChatVariation || showKayakoVariation );
+		const { details } = this.state.olark;
 
 		if ( ! details.isConversing ) {
 			analytics.tracks.recordEvent( 'calypso_help_offline_form_display', {
-				form_type: showKayakoVariation ? 'kayako' : 'forum'
+				form_type: 'kayako'
```

Comment:
Hmm, I'm not sure this follows. What if we change what eligibility for chat means? For example, we have greater availability to offer chat to users with subscriptions right now, but when we have capacity to expand that, I think we'd have a case of users who are not currently eligible for chat who might be in the future. Or, to try to outline it since this is sort of confusing, a future scenario might look like this:

1. A user who has no subscriptions loads the form.
2. We have capacity to offer chat to that user.
3. The user initiates a chat.
4. Operators go away.
5. We need to show an offline message.

Wouldn't we still need to record the forum event in this case?